### ISSUE TITLE
Improve SEO and accessibility

### ISSUE THREAD

**Issue by gomorizsolt** [c-hive/dotdev#141]
The page lacks meta tags to be crawled by search engines. I'd also pay attention to accessibility if possible (it's a bit huge topic but I'm happy to collect resources to get started with it).

**Comment by thisismydesign** [comment]
Thanks. This is a huge topic and not in focus right now. This page can serve more as a CV and less for marketing. For that purpose I'd recommend a site builder instead.
